src/main/resources/META-INF/plugin.xml

```diff
@@ -51,10 +51,10 @@
                     class="fi.aalto.cs.apluscourses.intellij.actions.APlusAuthenticationAction"
                     text="Set A+ Token"
                     description="Set an access token for submitting assignments to A+."/>
-            <action id="fi.aalto.cs.apluscourses.intellij.actions.ResetProjectAction"
-                    class="fi.aalto.cs.apluscourses.intellij.actions.ResetProjectAction"
-                    text="Reset A+ Project"
-                    description="Resets the open A+ course project."/>
+            <!--            <action id="fi.aalto.cs.apluscourses.intellij.actions.ResetProjectAction"-->
+            <!--                    class="fi.aalto.cs.apluscourses.intellij.actions.ResetProjectAction"-->
+            <!--                    text="Reset A+ Project"-->
+            <!--                    description="Resets the open A+ course project."/>-->
```

Comment:
Juha said some students used this feature to fix some issues. So, should we still keep it?

Another note: if we're getting rid of it, should the lines be removed instead of just commenting out and the class also be removed?